app.go

```diff
@@ -453,8 +476,22 @@ const (
 	DefaultReadBufferSize       = 4096
 	DefaultWriteBufferSize      = 4096
 	DefaultCompressedFileSuffix = ".fiber.gz"
+	DefaultLivenessEndpoint     = "/healthz"
+	DefaultReadinessEndpoint    = "/readyz"
 )
 
+var DefaultHealthFunction = func(c *Ctx) bool { return true }
+
+var ProbeCheckerHandler = func(checker ProbeChecker) Handler {
```

Comment:
I’d not export this, or what’s the benefit of doing so?

Reply:
All the other default values are exported, why not this function? Only the ProbeCheckerHandler shouldn't be exported in my opinon